src/handlers/inventory.ts

```diff
@@ -0,0 +1,72 @@
+import { Subscription } from "nats";
+import { NatsHandler, jsonCodec, getConnection } from "../nats";
+import { InventoryStorage } from "../contracts/inventory.types";
+import {
+    assign_item,
+    update_item
+} from "../contracts/inventory";
+import {
+    InventoryItem,
+    MichelsonInventoryItem,
+    JSONInventoryItem
+} from "../contracts/inventoryItem";
+import { MichelsonMap } from "@taquito/michelson-encoder";
+
+export const inventoryHandlers: NatsHandler[] = [
+    [
+        "tokenization-service_assign_item",
```

Comment:
we should refactor this service to follow the same conventions as the item-store: 

- tokenization-service is prefixed when registering the handlers
- the logic for calling the smart contract entrypoints should be moved into a services/inventoryContract.ts file